test/App.test.jsx

```diff
@@ -2,16 +2,13 @@ import React from 'react';
 import { configure, shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import toJson from 'enzyme-to-json';
-import 'dotenv/config';
 import App from '../src/App';
 
 configure({ adapter: new Adapter() });
 describe('<App />', () => {
   it('h1 contains correct text', () => {
     const wrapper = shallow(<App />);
-    expect(wrapper.find('h1').text()).toBe(
-      `Welcome to Naija Chop Chop ${process.env.ENV_VAR}`
-    );
+    expect(wrapper.find('h1').text()).toBe(`Welcome to Naija Chop Chop`);
```

Comment:
Why are we making this change?

Reply:
Still waiting for an answer to this

Reply:
😕

Reply:
I have replied to this conversation before.

[Answer] We agreed to remove it because we thought it was the cause of the deployment error.